doc/en/mark.rst

```diff
@@ -42,8 +43,13 @@ Marks can be registered in ``pytest.ini`` like this:
         slow
         serial
 
-This can be used to prevent users mistyping mark names by accident. Test suites that want to enforce this
-should add ``--strict`` to ``addopts``:
+When the ``--strict`` command-line flag is passed, any unregistered marks
+applied with the ``@pytest.mark.name_of_the_mark`` decorator will trigger an
+error, including built-in marks such as ``skip`` or ``xfail``.
```

Comment:
> including built-in marks such as ``skip`` or ``xfail``.

What do you mean by this sentence? I mean, those builtin marks will never raise an error.

Reply:
Whoops, yeah - they're explicitly registered by their plugins.  Will edit.